nautobot/extras/templates/extras/inc/jobresult.html

```diff
@@ -1,111 +1,65 @@
-                    <td>Status</td>
-                    <td><span id="pending-result-label">{% include 'extras/inc/job_label.html' with result=result %}</span></td>
-                </tr>
-                <tr>
-                    <td>Started at</td>
-                    <td>{{ result.created }}</td>
-                </tr>
-                <tr>
-                    <td>User</td>
-                    <td>{{ result.user }}</td>
-                </tr>
-                <tr>
-                    <td>Duration</td>
-                    {% if result.completed %}
-                        <td>{{ result.duration }}</td>
-                    {% else %}
-                        <td><img id="pending-result-loader" src="{% static 'img/ajax-loader.gif' %}"></td>
-                    {% endif %}
-                </tr>
-
-                {% if result.completed %}
-                    {% for grouping, data in result.data.items %}
-                        {% if grouping != "total" and grouping != "output" %}
-                            <tr>
-                                <td><code><a href="#{{ grouping }}">{{ grouping }}</a></code></td>
-                                <td class="text-right report-stats">
-                                    <label class="label label-success">{{ data.success }}</label>
-                                    <label class="label label-info">{{ data.info }}</label>
-                                    <label class="label label-warning">{{ data.warning }}</label>
-                                    <label class="label label-danger">{{ data.failure }}</label>
-                                </td>
-                            </tr>
-                        {% endif %}
-                    {% endfor %}
-                {% endif %}
-            </table>
-        </div>
-        {% plugin_left_page result %}
-    </div>
-    <div class="col-md-6">
-        {% include 'inc/custom_fields_panel.html' %}
-        {% plugin_right_page result %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Summary of Results</strong>
     </div>
+    <table class="table table-hover panel-body">
+        <tr>
+            <td>Status</td>
+            <td><span id="pending-result-label">{% include 'extras/inc/job_label.html' with result=result %}</span></td>
+        </tr>
+        <tr>
+            <td>Started at</td>
+            <td>{{ result.created }}</td>
+        </tr>
+        <tr>
+            <td>User</td>
+            <td>{{ result.user }}</td>
+        </tr>
+        <tr>
+            <td>Duration</td>
+            {% if result.completed %}
+                <td>{{ result.duration }}</td>
+            {% else %}
+                <td><img id="pending-result-loader" src="{% static 'img/ajax-loader.gif' %}"></td>
+            {% endif %}
+        </tr>
+    </table>
+</div>
+{% plugin_left_page result %}
+<div class="col-md-6">
+{% include 'inc/custom_fields_panel.html' %}
+{% plugin_right_page result %}
 </div>
```

Comment:
While you're cleaning things up here, this block (from `plugin_left_page` to `</div>`) should, I think, be in the base `extras/jobresult.html` page, not in this `inc` file.

Reply:
Per our discussion on Slack, this matches the pattern in the existing templates here, so we can keep it as-is for backward compatibility.